### PR TITLE
example: add time-series models with statsmodels

### PR DESCRIPTION
**Pull request by jsamantaucd**
Feature request
I have implemented and tested a feature that trains a time series model - ARIMA and serves it using BentoML's custom runner to make predictions.

Motivation
I believe it would be a nice guide to someone exploring time series models with custom runner implementation for models like ARIMA.

Others:
This feature was tested on Bentoml v1.2.16

### CODE REVIEW

**Comment by jsamantaucd** [comment]
Hi @xianml, as per your suggestion I have created the same arima model example using bentoml >=1.2, please help to review. Regards!

File: examples/arima-model/train.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    # Calculate root mean squared error
+    rmse = np.sqrt(mean_squared_error(y_test, y_pred))
+    print("Root Mean Squared Error:", rmse)
+
+    # Save model with BentoML
+    saved_model = bentoml.picklable_model.save_model(
+        "arima_forecast_model",
+        model,
+        signatures={"predict": {"batchable": True}},
+    )
```
Comment (jianshen92) [inline examples/arima-model/train.py:52]: The recommended API here is using `bentoml.models.create` with our latest API. https://docs.bentoml.org/en/latest/guides/model-store.html

**Comment by frostming** [comment]
Hi @jsamantaucd We are preparing to restructure the examples and do not accept new framework-specific examples.

You can make it in a standalone repository and we can link it in our README, thanks.

**Comment by jsamantaucd** [comment]
Hi @frostming and @jianshen92, thank you for your valuable feedback, I incorporated everything in the current stats model example and created a standalone repository here: https://github.com/jsamantaucd/BentoStatsmodel, please kindly link it in your README, thanks!

**Pull request closed without merging**